docker/docker.go

```diff
@@ -100,8 +107,19 @@ if [ "$BUILD_TYPE" = "coverage" ]; then
 fi
 `
 
+// tcDisabledString returns an empty string is increasing the latency using
```

Comment:
maybe:
"tcDisabledString returns an empty string is if the tc utility is configured to increase the latency, or a comment otherwise"